tests/baselines/reference/tsxElementResolution10.errors.txt

```diff
@@ -24,7 +24,7 @@ tests/cases/conformance/jsx/file.tsx(19,2): error TS2322: Type '{ x: number; ren
      ~~~~
 !!! error TS2786: 'Obj1' cannot be used as a JSX component.
 !!! error TS2786:   Its instance type '{ x: number; }' is not a valid JSX element.
-!!! error TS2786:     Property 'render' is missing in type '{ x: number; }' but required in type 'ElementClass'.
+!!! error TS2786:     The 'render' attribute is missing, but is required for this 'Obj1' tag.

```

Comment:
Is this one right @weswigham?

Reply:
....was it ever right?

Reply:
The original error was correct - look at the root; this expression isn't a valid tag (it's not renderable), so we're not comparing attributes here. This message shouldn't have changed.

Reply:
Ah, got it.

And another question, 
```
var o = {
	f: () => (<> </>)
}
<o.f /> // what is the tag name of this element?  `o.f` or just `f`?
```

Reply:
Another question:
```
class A {
	jsx() {
		return (<></>)
	}

	useJsx() {
		return (<this.jsx />) // what is this tag?

	}
}
```